src/Uno.Extensions.Reactive/Core/Internal/StateStore.None.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Uno.Extensions.Reactive.Core;
+
+internal class NoneStateStore : IStateStore
+{
+	/// <inheritdoc />
+	public TState GetOrCreateState<TSource, TState>(TSource source, Func<SourceContext, TSource, TState> factory)
+		where TSource : class
+		where TState : IStateImpl, IAsyncDisposable
+		=> throw new InvalidOperationException("Cannot create a state on SourceContext.None");
```

Comment:
Is this something that may happen in normal cases? Is it something the user needs to troubleshoot?

Reply:
This might happen only when hacking the system, definitely not in standard usage. I'll however add a more detailed message (among with the error message in the `SourceContext` above) to explain how to deal with that: https://github.com/unoplatform/uno.extensions/commit/550ddaf2156bfdcc0901347740dc90f011051a16 (Might push it only in next PR depending of the result of the current build)